static/src/javascripts-legacy/projects/common/modules/experiments/tests/paid-content-vs-outbrain.js

```diff
@@ -24,12 +24,20 @@ define([
         variants: [
             {
                 id: 'paid-content',
-                test: function () {
+                // TODO: ask someone about this
```

Comment:
I'd say we should remove it for now, we can always re-introduce it at a later date.